### ISSUE TITLE
Shortcut not recognising input is coming from different peripherals

### ISSUE THREAD

**Issue by PromeThaeus** [mumble-voip/mumble#5477]
### Description

Using a twin Thrustmaster T16000M FCS setup, setting a shortcut for Push-to-Talk on one stick reads input from both sticks (i.e. setting the PTT shortcut to be the left stick's trigger causes the right stick's trigger to also function as a PTT shortcut).

### Steps to reproduce

1. Open the Configure dropdown.
2. Open the Settings window.
3. Navigate to the Shortcuts menu.
4. Add a PTT shortcut and set it to left joystick trigger, reading '[044F:B10A]:0'.
5. Test shortcut by depressing the left joystick trigger (works).
6. Test shortcut by depressing the right joystick trigger (also works).

### Mumble version

1.4.230

### Mumble component

Client

### OS

Windows

### Reproducible?

Yes

**Comment by PromeThaeus** [comment]
Further testing indicates that the Mumble application is assigning the same identifier number ('[044F:B10A]:0') to each individual T16000M FCS joystick. It successfully assigns a different identifier number to a completely different peripheral (e.g. XBox controller).

**Comment by Krzmbrzl** [comment]
Did you verify that Windows itself is detecting these as individual devices? Do these buttons work independently in other applications?

**Comment by PromeThaeus** [comment]
Yes, Windows is detecting them as separate devices, and they work independently in other applications (e.g. flight simulators).

**Comment by Krzmbrzl** [comment]
@davidebeatrici do you have an idea about this?

**Comment by davidebeatrici** [comment]
The relevant code:

https://github.com/mumble-voip/mumble/blob/db3a280585ae0fb7798f18ba6487644e0b97802d/src/mumble/GlobalShortcut_win.cpp#L565-L622

@PromeThaeus Do you perhaps see the full device name if you edit the shortcut?

**Comment by PromeThaeus** [comment]
Edit shortcut menu reads the device as "[044F:B10A] Thrustmaster" regardless of which device is being used.
